src/base/bittorrent/session.h

```diff
@@ -330,6 +334,16 @@ namespace BitTorrent
         void setDiskCacheTTL(uint ttl);
         bool useOSCache() const;
         void setUseOSCache(bool use);
+        bool guidedReadCache() const;
```

Comment:
@glassez:  

>Even `guidedReadCache` can be used as bool property name. 

Then I must have misunderstood the nature of your request.  What exactly are you asking to be changed?